superset-frontend/src/dashboard/components/dnd/DragDroppable.jsx

```diff
@@ -77,6 +77,7 @@ const defaultProps = {
 const DragDroppableStyles = styled.div`
   ${({ theme }) => css`
     position: relative;
+    transform: translate3d(0, 0, 0);
```

Comment:
Could you add a comment with a link to the comment in react-dnd repo that provided this solution?